libraries/AP_Notify/Buzzer.cpp

```diff
@@ -32,6 +33,7 @@ bool Buzzer::init()
 
     // setup the pin and ensure it's off
     hal.gpio->pinMode(BUZZER_PIN, HAL_GPIO_OUTPUT);
+    
```

Comment:
Cosmetic changes should be done in a separate commit, but these two don't seem necessary, leftovers?